harness/determined/pytorch/_pytorch_trial.py

```diff
@@ -29,14 +29,40 @@
     pass
 
 
+class TrainUnit:
+    def __init__(self, value):
+        self.value = value
```

Comment:
drop the superclass's `__init__`, just have each class use `self.value = value`.  One less line of code overall, and also more straightforward.

Basically, anything to do with `super()` is harder to think about than anything that doesn't require `super()`.  This is sort of a trivial instance of that, I suppose.

Reply:
Well, or you could go the other way and have `Epoch` and `Batch` not even define `__init__()`.  That would be closer to what you currently have, while still avoiding any `super()` mechanics.

Reply:
went with the second approach since it seems more straightforward.